### PR TITLE
Added update and anonymisation functionality to interpreters/upload

### PR DESCRIPTION
**Pull request by will-freshworks**
- Also got e2e BE tests working again.
- Also added a couple of languages as per CSBISS-49

### CODE REVIEW

**Comment by will-freshworks** [comment]
Note: I am aware that `faker-js` exists but I needed some very specific logic/behaviour for the fake data, so I just wrote it myself.

File: api/src/interpreter/interpreter.service.ts [hunk 1]
```diff
@@ -1,16 +1,16 @@
 import { Injectable } from '@nestjs/common';
 import { InjectRepository } from '@nestjs/typeorm';
-import { Repository } from 'typeorm';
-
 import { BookingDateEntity } from 'src/booking/entities/booking-date.entity';
+import { BookingPeriod } from 'src/booking/enums/booking-period.enum';
 import { SuccessResponse } from 'src/common/interface/response/success.interface';
+import { Repository } from 'typeorm';
```
Comment (jerryfreshworks) [inline api/src/interpreter/interpreter.service.ts:6]: curios why moving this from line 3, we usually put npm package on the top.

**Review by jerryfreshworks — APPROVED** [review]
Great work! Good type like `Record<K, T>`, good testing, and interesting random generator.

File: api/src/interpreter/interpreter.service.ts (hunk 1 above)
Comment (will-freshworks) [reply api/src/interpreter/interpreter.service.ts:6]: No idea. Possibly something vscode did automatically when I ran Shift + Option + F.

**Pull request merged**